core/src/main/java/com/linecorp/armeria/internal/client/dns/DnsUtil.java

```diff
@@ -121,5 +121,9 @@ public static int defaultNdots() {
         return DEFAULT_NDOTS;
     }
 
+    public static long defaultDnsQueryTimeoutMillis() {
+        return 5000; // 5 seconds
+    }
```

Comment:
Can we use a static final field instead of a method so we can use `{@value ..}` when mentioning the default?